templates/test/ci/cluster-template-prow-external-cloud-provider.yaml

```diff
@@ -240,3 +243,17 @@ spec:
     namespace: ${AZURE_CLUSTER_IDENTITY_SECRET_NAMESPACE}
   tenantID: ${AZURE_TENANT_ID}
   type: ServicePrincipal
+---
+apiVersion: addons.cluster.x-k8s.io/v1beta1
+kind: ClusterResourceSet
+metadata:
+  name: ${CLUSTER_NAME}-calico-windows
```

Comment:
are there windows nodes in this template?

Reply:
No, but this change should be reverted now.